src/Components/Navs.mdx

```diff
@@ -83,3 +84,40 @@ They can be `justified` to take the full width of their parent.
     </NavDropdown>
   </Nav>
 </Playground>
+
+## Nav underline style ![Not Ready Badge](https://img.shields.io/badge/component-not%20ready-red.svg#badge)
+
+Underline styled tabs can be build with Router, li, and Navlink elements by using "nav nav-tabs-underline" on a parent and an li element containing a NavLink or Link element.
+
+<Playground>
+  <Router>
```

Comment:
We don't need Router / div / ul here. Instead it is preferred to follow the React-Bootstrap API:

```
<Nav className="nav-slider-wrapper">
  <NavItem active>Tab 1</NavItem>
  <NavItem>Tab 2</NavItem>
  <NavItem>Tab 3</NavItem>
</Nav>
```

https://5c507d49471426000887a6a7--react-bootstrap.netlify.com/components/navs/#navs-dropdown

Reply:
See above, but I'll give it a shot.

Reply:
done